sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -61,7 +60,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultScheduler.class);
 
-    protected final ExecutorService executor = Executors.newFixedThreadPool(1);
+    /**
+     * 1 Thread for processing offers off the queue in {@link #processOffers()}
```

Comment:
With this structure, both threads could be used by a single operation (e.g. two concurrent processOffers() calls taking up the whole pool). Would it make sense to have two isolated queues?